<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizClass</class>
 <widget class="QMainWindow" name="QuizClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quiz</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="questions_list_widget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_line_edit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>660</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="answer_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="score_line_edit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>350</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>300</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1358</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Quiz.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>answer_button</sender>
   <signal>clicked()</signal>
   <receiver>QuizClass</receiver>
   <slot>answer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>720</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>738</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>questions_list_widget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>QuizClass</receiver>
   <slot>enable_answer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>answer()</slot>
  <slot>enable_answer()</slot>
 </slots>
</ui>
